tests/e2e/test_requests.py

```diff
@@ -221,13 +230,16 @@ def request_fence(method, coordinates, coordinate_system):
 
     multipart_data = decoder.MultipartDecoder.from_response(rdata)
     assert len(multipart_data.parts) == 2
-    metadata = multipart_data.parts[0].content
+    metadata = json.loads(multipart_data.parts[0].content)
     data = multipart_data.parts[1].content
 
-    assert len(metadata) == 0
+    shape = (
```

Comment:
`shape = tuple(metadata["shape"])`

Or maybe numpy even accept the list directly? Without the tuple cast?